data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"<first>! I found a publishing company who's willing to publish my book! I sent them a digital copy of my book and they have instructed me to visit their office on <destination>!", exclaims Anna, once again in an enthusiastic tone.`
+			choice
+				`	"What's the publishing company called?"`
+			`	"They're called 'Kimek Reader Publishing'.`
+			choice
+				`	"Great!"`
+				`	"Let's go!"`
+			`	Anna closes her laptop, places her things in her bag, and begins walking to your ship. You and Anna enter the ship, as she settles into her bunk. It's time to go to <destination>, where the main offices of 'Kimek Reader Publishing' are located.`
+				accept
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive', her novel about humans stealing jump drives from the Quarg. She hastily opens the hatch as soon as it unlocks.`
+			choice
+				`	"Wait for me!"`
+			`	Anna hears your scream and stops. "We're here, Anna. Now, we need to find their offices.", you say.`
+			`	"That won't be necessary. In the message they sent me, they included precise instructions on how to reach their offices."`
+			`	You begin following Anna to the publisher's offices. You walk through endless rows of shops and streets, take a train to a large white building, and eventually find what looks like the Kimek equivalent of an elevator.`
+			`	You and Anna enter the elevator, as Anna presses a button that closes the door and takes you up. When the door opens, you see a Kimek in front of you, a necklace around it with ornate jewelry and a translation device.`
+			`	"Greetings, humans. Follow me, you must, so publish your book, we can.", says the Kimek through its translation box.`
+			`	At the direction of the Kimek, you and Anna begin walking until directed to a room with glass walls. Anna puts down the copy of her book on the table, and the three of you take a seat.`
+			choice
+				`	"Hello. My name is <first> <last>, and this is my friend Anna Fangvin."`
+			`	"Greetings. At Kimek Reader Publishing, I work. A human, I have never encountered. But the book you sent, translate it we were able to."`
+			`	"Read it, we still haven't, and run it past the Heliarch censors, we must."`
+			`	Two Heliarch agents enter the room. "Supervise the interview, we are obligated to."`
+			choice
+				`	"How long will I have to wait?"`
+					goto "how long"
+				`	"Should I stay here to accompany Anna?`
+					goto "should I accompany anna"
+			label "how long"
+			`	"Wait a few hours, you must. But quick, this will be.", says the Kimek, interviewing you.`
+			choice
+				`	"Should I stay here to accompany Anna?"`
+			label "should I accompany anna"
+			`	"To accompany her in this room, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
```

Comment:
So if I got this right what they're saying is that the player can be present for the censoring process if they wish to, but they're also free to leave for the lounge. If that is indeed correct then I think you should make this a bit clearer, as it does seem confusing at first: you're told about being allowed to accompany her to the room, but that has already happened, you're at the room, so you've already accompanied her there. That tells me that probably next thing they'll tell me is something like "however, you can't be present for the process, so you'll need to wait" which does seem supported by the mention of having a lounge. However, the next sentence says that the player can leave if they do find the process boring.
I'd rephrase this entirely to be more akin to them starting out by saying that you can stay with her if you want, but you aren't required to be present for the process, so you may also leave at any time if you'd rather wait it out somewhere.

Reply:
15b5570a9d8f60f3e768dba52013d58f5c48363b